data/CentOS-9.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Since 2ef9d94c2e0ec388a5c1013d2c0148719a6edcca I think this file is redundant.

Reply:
A thanks, indeed. I'll leave only one.